### ISSUE TITLE
build fails at fresh template "Cannot find namespace 'JSX'."

### ISSUE THREAD

**Issue by vogsphar** [preactjs-templates/typescript#10]
```bash
~ $ npm i -g preact
~ $ preact create typescript t
~ $ cd t
~/t $ npm run build

> t@0.0.0 build /playground/t
> preact build

 Build  [=================== ] 95% (2.1s) emitting
 ssr-bundle.06ab0.css ⏤  617 B (+617 B)
        ssr-bundle.js ⏤  4.07 kB (+4.07 kB)

✖ ERROR ./node_modules/preact-render-spy/index.d.ts
ERROR in ./node_modules/preact-render-spy/index.d.ts(98,23):
TS2503: Cannot find namespace 'JSX'.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! t@0.0.0 build: `preact build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the t@0.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /playground/.npm/_logs/2020-01-29T11_53_05_391Z-debug.log

~/t $ node --version
v12.14.1
~/t $ preact --version
         ▄▄
     ▄▄▓▓▓▓▓▓▄▄
  ▄█▀▀█▓▓▓▓▓▓▓▀▀█▄▄
▐▓▌▐▓▓▓▒▄ ▀▄▄▓▓▓▌▐▓▌
▐▓▓▄▀▓▀ ▄▓▓▄▄▀▓▓ ▓▓▌
▐▓▓▓▌ ▒▓▌  ▐▓▓  ▓▓▓▌ preact-cli 2.2.1
▐▓▓ ▒▓▄▄▀▓▓▀ ▄▓▓ ▓▓▌
▐▓▌▐▓▓▓▀▀▄▄▀▀▓▓▓▌▐▓▌
  ▀█▄▄▒▓▓▓▓▓▓▒▄▄▒▀
      ▀▓▓▓▓▓▓▀▀
         ▀▀
```

**Comment by mknj** [comment]
Workaround: `rm src/tests/header.test.tsx`

then the ERROR vanishes and we only see some warnings

```bash
$ npm run build
⚠ WARN Conflict: Multiple assets emit different content to the same filename bundle.55000.css
⚠ WARN Conflict: Multiple assets emit different content to the same filename route-home/index.tsx.chunk.80572.css
⚠ WARN Conflict: Multiple assets emit different content to the same filename route-profile/index.tsx.chunk.f25b6.css
⚠ WARN Conflict: Multiple assets emit different content to the same filename bundle.55000.css
⚠ WARN Conflict: Multiple assets emit different content to the same filename route-home/index.tsx.chunk.80572.css
⚠ WARN Conflict: Multiple assets emit different content to the same filename route-profile/index.tsx.chunk.f25b6.css
```

**Comment by RyanChristian4427** [comment]
The issues with render spy were (at least temporarily) fixed by #13.

**Comment by ForsakenHarmony** [comment]
I'll close this for now, let us know if you run into it again
